src/widgets/PrivacyWidget.php

```diff
@@ -0,0 +1,175 @@
+<?php
+namespace luya\privacy\widgets;
+
+use Yii;
+use yii\web\Cookie;
+use luya\base\Widget;
+use luya\cms\helpers\Url;
+use luya\helpers\Json;
+use luya\privacy\Module;
+use luya\privacy\assets\PrivacyWidgetAsset;
+use luya\privacy\traits\PrivacyTrait;
+
+/**
+ * Privacy Widget
+ * 
+ * This widget will show a privacy notification. It includes a privacy (cookie) message, which can be accepted.
+ * 
+ * ```php
+ * PrivacyWidget::widget();
+ * ```
+ *
+ * @author Alex Schmid <alex.schmid@stud.unibas.ch>
+ * @since 1.0.0
+ * 
+ * @todo Widget id must be better
+ * @todo Position -> fixed / relative
+ */
+class PrivacyWidget extends Widget
+{
+    use PrivacyTrait;
+
+    const PRIVACY_COOKIE_NAME = '_privacyPolicy';
+
+    /**
+     * @inheritdoc
+     */
+    public function getViewPath()
+    {
+        return '@privacy/views/widgets';
```

Comment:
You don't need to define a view path. This is should work out of the box for widgets.